src/Orleans/Streams/PersistentStreams/StreamQueueBalancerType.cs

```diff
@@ -1,19 +1,22 @@
 namespace Orleans.Streams
 {
-    public enum StreamQueueBalancerType
+    /// <summary>
+    /// Built-in stream queue balancer type which is supported natively in orleans
+    /// </summary>
+    public static class BuiltInStreamQueueBalancerType
```

Comment:
Changing the name of this class will make it a breaking change. But I slightly prefer this name, which has clearer indication, especially now we support pluggable queue balancer now. But I will be fine if you want to change it back